homeassistant/components/connector/const.py

```diff
@@ -0,0 +1,32 @@
+"""Constants for the Connector integration."""
+from homeassistant.const import Platform
+
+DOMAIN = "connector"
+DEFAULT_HUB_NAME = "Connector Hub"
+MANUFACTURER = "Connector Shades"
+DEFAULT_GATEWAY_NAME = "Connector Shades"
+
+PLATFORMS = [Platform.COVER, Platform.SENSOR]
```

Comment:
So you reference the platforms here but don't use them in init.py?

Reply:
Thank you for your reminder, I will modify it